app/src/main/java/edu/uc/reedws/musiclink/ui/main/MainFragment.kt

```diff
@@ -12,7 +12,8 @@ class MainFragment : Fragment() {
     private lateinit var viewModel: MainViewModel
 
     override fun onCreateView(
-        inflater: LayoutInflater, container: ViewGroup?,
+        inflater: LayoutInflater,
+        container: ViewGroup?,
```

Comment:
Just a small housekeeping/best practice change -- Overall, you guys had really great code this sprint, so most of my changes are very small things. You guys did great!! Thanks!